models/DataObject/ClassDefinition/Data/StructuredTable.php

```diff
@@ -609,4 +602,26 @@ public function isEqual($oldValue, $newValue): bool
 
         return $this->isEqualArray($oldData, $newData);
     }
+
+    public function getParameterTypeDeclaration(): ?string
+    {
+        return '?\\Pimcore\\Model\\DataObject\\Data\\StructuredTable';
+    }
+
+    public function getReturnTypeDeclaration(): ?string
+    {
+        return '?\\Pimcore\\Model\\DataObject\\Data\\StructuredTable';
+    }
+
+    public function getPhpdocInputType(): ?string
+    {
+        return '\\Pimcore\\Model\\DataObject\\Data\\StructuredTable';
```

Comment:
The parameter type is nullable and should match.
```suggestion
        return '\\Pimcore\\Model\\DataObject\\Data\\StructuredTable|null';
```